app/plugin_api_test.go

```diff
@@ -469,6 +471,7 @@ func TestPluginAPILoadPluginConfiguration(t *testing.T) {
 			}
 		]
 	}}`, "testloadpluginconfig", th.App)
+
```

Comment:
nit: can you revert these lines just to avoid unrelated diffs?

Reply:
solved by a24cae9